src/services/apiService.js

```diff
@@ -239,9 +248,10 @@ angular.module('openshiftCommonServices')
         continue;
       }
       return {
+        resource: resource.resource,
+        version:  resource.version,
         hostPort: api.hostPort,
         prefix:   api.prefix,
-        version:  resource.version,
         namespaced: discoveredResource.namespaced
```

Comment:
why arent you getting the verbs here as well, im looking at the api discovery on my machine and i get verbs on these